examples/next-typescript/pages/select.tsx

```diff
@@ -0,0 +1,76 @@
+import { Container, Select } from "@tonightpass/kitchen";
+import { NextPage } from "next";
+import React from "react";
+import { BsArrowUpCircle } from "react-icons/bs";
+
+const SelectPage: NextPage = () => {
+  return (
+    <>
+      <Container
+        gap={10}
+        style={{
+          maxWidth: 700,
+          margin: "0 auto",
+        }}
+      >
+        <p>sizes</p>
+        <Container gap={10} row style={{ alignItems: "center" }}>
+          <Container row>
+            <Select size="small" placeholder="Small" />
+          </Container>
+          <Container row>
+            <Select size="normal" placeholder="Default" />
+          </Container>
+          <Container row>
+            <Select size="large" placeholder="Large" />
+          </Container>
+        </Container>
+
+        <p>sizes, prefix and suffix</p>
+        <Container gap={10} row style={{ alignItems: "center" }}>
+          <Container row>
+            <Select
+              size="small"
+              placeholder="Small"
+              prefix={BsArrowUpCircle}
```

Comment:
First of all you use icons that are not RemixIcon and as you know we only use RemixIcon.